src/ngraph/runtime/gpu/gpu_backend.hpp

```diff
@@ -59,6 +67,17 @@ namespace ngraph
                 std::vector<PerformanceCounter>
                     get_performance_data(std::shared_ptr<Function> func) const override;
 
+                class BackendContext
+                {
+                public:
+                    BackendContext();
+                    ~BackendContext();
+                    cublasHandle_t m_cublas_handle;
```

Comment:
Since `m_cublas_handle` and `m_cudnn_handle` are already in `m_ctx`, looks redundant here for me.

Reply:
Good point, they are removed and allocated on the heap, the pointers for which are stored in the runtime context.